examples/http/cisco_meraki_channel_utilisation.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Cisco Meraki Channel Utilisation
+ * Description: This script extracts channel utilization information from Cisco Meraki networks using the Meraki Dashboard API.
+ * 
+ * Communication protocol is HTTPS
+ * 
+ * Tested on Cisco Meraki version wireless-25-13 
+ *
+ * Creates a Custom Driver table with the following columns:
+ *      - Network: Network name 
+ *      - Device: Device model
+ *      - Channel: Channel id 
+ *      - Channel Utilization: Percentage of total channel utiliation for the given radio
+ *      - Wifi Utilization: Percentage of wifi channel utiliation for the given radio
+ *      - Non Wifi Utilization: Percentage of non-wifi channel utiliation for the given radio
+ *      - Start Timestamp: The start time of the channel utilization interval
+ *      - End Timestamp: The end time of the channel utilization interval
+ * 
+ **/
+
+var device = D.createExternalDevice("api.meraki.com"); 
+
+// The Id of the organization, obtained from the Cisco Meraki dashboard
+var organizationId = D.getParameter("organizationID"); 
+
+// If networkId is 'ALL', it returns all network IDs for the given organization.
+// Otherwise, it resolves with the specified networkId
+var networkId = D.getParameter("networkID"); // The ID of the network
+
+// The API key 
+var apiKey = D.getParameter("apiKey"); 
+
+// Table to store channel utilization data 
+var table = D.createTable(
+    "Channel Utilization",
+    [
+        { label: "Network", valueType: D.valueType.STRING },
+        { label: "Device", valueType: D.valueType.STRING },
+        { label: "Channel", valueType: D.valueType.STRING },
+        { label: "Channel Utilization", unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Wifi Utilization",  unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Non Wifi Utilization",  unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Start Timestamp", valueType: D.valueType.DATETIME },
```

Comment:
the start and end timestamps are a bit strange to have, I would suggest removing the columns